server/chat-commands.js

```diff
@@ -655,7 +655,7 @@ const commands = {
 	unblockpmshelp: [`/unblockpms - Unblocks private messages. Block them with /blockpms.`],
 
 	status(target, room, user, connection, cmd) {
```

Comment:
```suggestion
	'!status': true,
	status(target, room, user, connection, cmd) {
```

I think this is also necessary for setting status in the global room.

Reply:
Why? It worked when I tested it

Reply:
It makes the command usable in PMs, and commands in the global room are implemented as PMs to the server.

Even if some other weirdness makes it optional, there's no reason _not_ to add this.